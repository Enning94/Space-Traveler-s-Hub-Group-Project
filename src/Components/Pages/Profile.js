import RocketProfile from '../RocketProfile';

const Profile = () => (
  <main className="grid sm:grid-cols-2 grid-cols-1">
    <div>Missions</div>
    <div>
      <RocketProfile />
    </div>
  </main>
);

export default Profile;
