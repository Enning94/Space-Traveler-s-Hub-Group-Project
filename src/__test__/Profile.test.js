import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import Profile from '../Components/Pages/Profile';
import store from '../Redux/configureStore';

describe('Renders correctly', () => {
  it('Renders page correctly', () => {
    const { container } = render(<Provider store={store}><Profile /></Provider>);
    expect(container).toMatchSnapshot();
  });
});
