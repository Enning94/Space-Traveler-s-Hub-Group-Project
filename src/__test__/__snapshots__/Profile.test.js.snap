// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly Renders page correctly 1`] = `
<div>
  <main
    class="grid sm:grid-cols-2 grid-cols-1"
  >
    <div>
      <div
        class="pt-3"
      >
        <h2
          class="text-xl font-semibold font-important"
        >
          No Missions yet.
        </h2>
        <div>
          <ul
            class="grid grid-cols-1 my-4 mx-2"
          />
        </div>
      </div>
    </div>
    <div>
      <div
        class="pt-3"
      >
        <h2
          class="text-xl font-semibold font-important"
        >
          No Rockets yet.
        </h2>
        <ul
          class="grid grid-cols-1 my-4 mx-2"
        />
      </div>
    </div>
  </main>
</div>
`;
